.ci/bump-version.sh

```diff
@@ -11,12 +11,13 @@ docker run --rm -t \
   -w /app \
   -v "$(pwd):/app" \
   node:12-alpine /bin/sh -c "set -x
-    CI=true npm install elastic-apm-node@${AGENT_VERSION}"
+    CI=true npm install --ignore-scripts elastic-apm-node@${AGENT_VERSION}"
 
 
 ## Bump agent version in the Dockerfile
 sed -ibck "s#\(org.label-schema.version=\)\(\".*\"\)\(.*\)#\1\"${AGENT_VERSION}\"\3#g" Dockerfile
+rm -f Dockerfilebck
```

Comment:
Question: Where does `Dockerfilebck` come from?

Reply:
From the `sed -ibck ...` option:

```
     -I extension
             Edit files in-place, saving backups with the specified extension.  ...
             That can lead to unexpected
             results in many cases of in-place editing, where using -i is desired.

     -i extension
             Edit files in-place similarly to -I, but treat each file independently from other
             files. ...
```